### PR TITLE
use default_rowset_type to replace compaction_rowset_type

### PR DESCRIPTION
**Pull request by kangpinghuang**
to resolve the ambiguity of the rowset type configuration, replace compaction_rowset_type with default_rowset_type.

### CODE REVIEW

File: docs/documentation/cn/administrator-guide/segment-v2-usage.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+	`show alter table rollup;`
+
+	看到会有一个名字为__v2_table_name的rollup表正在创建。
+
+3. 通过查询base表和segment v2格式的rollup表，验证查询结果是否一致（查询可以来自audit日志）
+
+	为了让查询使用segment v2的rollup表，增加了一个session变量，通过在查询中使用如下的语句，来查询segment v2格式的index：
+
+	set use_v2_rollup = true;
```
Comment (kangkaisen) [inline docs/documentation/cn/administrator-guide/segment-v2-usage.md]: Why you strong coupling segment v2 and use_v2_rollup?
I think use_v2_rollup isn't related  with use_v2_rollup.

File: docs/documentation/cn/administrator-guide/segment-v2-usage.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+### 全量开启segment v2
+
+有两个方式：
+1. fe中有一个全局变量，通过设置全局变量default_rowset_type，来设置BE中使用segment v2作为默认的存储格式，命令如下：
+
+	set global default_rowset_type = beta;
+
+	使用这个方式，只需要执行上述命令，之后等待10s左右，FE会将配置同步到BE中。不需要在每个BE中进行配置。推荐使用这个配置。不过该方式目前没有简单的方式来验证BE上的default rowset type是否已经变更，一种办法是建一个表，然后查看对应的表的元数据。但是这样子比较麻烦，后续看需要可以在某个地方实现状态查看。
+
+2. 修改BE中的配置文件中的配置default_rowset_type和compaction_rowset_type为BETA。这种方式需要在每个BE中添加对应的配置。
```
Comment (kangkaisen) [inline docs/documentation/cn/administrator-guide/segment-v2-usage.md]:
```suggestion
2. 修改BE中的配置文件中的配置default_rowset_type为BETA。这种方式需要在每个BE中添加对应的配置。
```

File: docs/documentation/cn/administrator-guide/segment-v2-usage.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+- 导入延时
+- page cache命中率
+- string类型字段压缩率
+- bitmap索引的性能提升率和空间占用
+
+### 全量开启segment v2
+
+有两个方式：
+1. fe中有一个全局变量，通过设置全局变量default_rowset_type，来设置BE中使用segment v2作为默认的存储格式，命令如下：
```
Comment (chaoyli) [inline docs/documentation/cn/administrator-guide/segment-v2-usage.md]: I have a scene. After you set default_rowset_type, if BE will create alpha rowset instantly.
If you have a tablet A, B, C. You will have three different rowset type. How to handle it ?

**Review by chaoyli — CHANGES_REQUESTED** [review]

File: docs/documentation/cn/administrator-guide/segment-v2-usage.md (hunk 1 above)
Comment (kangpinghuang) [reply docs/documentation/cn/administrator-guide/segment-v2-usage.md]: it is for online upgrade process to validate the correctness of segment v2 by comparing the same query against the v1 and v2.

File: docs/documentation/cn/administrator-guide/segment-v2-usage.md (hunk 3 above)
Comment (kangpinghuang) [reply docs/documentation/cn/administrator-guide/segment-v2-usage.md]: It is compatible.

**Review by chaoyli — APPROVED** [review]

**Pull request merged**